components/gallery/GalleryItem.vue

```diff
@@ -194,8 +191,6 @@ import { resolveMedia } from '@/utils/gallery/media'
 import UnlockableTag from './UnlockableTag.vue'
 import { usePreferencesStore } from '@/stores/preferences'
 
-const NuxtImg = resolveComponent('NuxtImg')
```

Comment:
why remove `NuxtImg`?

Reply:
It is not needed if loading the original image. The content was from r2, not from cf-images. Unless we setup the original public on cf-images

Reply:
removing on carrousel img too?

Reply:
yes, carousel in gallery item is for rmrk item that has resources